src/components/appcontent.tsx

```diff
@@ -86,7 +86,7 @@ const AppContent: FC<Props> = ({ showSidePane, showNavbar, appContent }) => {
           <div
             className={`w-full ${
               showSidePaneGrid ? "hidden" : "block"
-            } sm:ml-[308px]  ${
+            } mr-[2vw] sm:ml-[27vw] lg:ml-[23vw] xl:ml-[23vw]  ${
```

Comment:
xl can be removed as xl and md values are same

Reply:
Resolved in latest commit.